src/org/traccar/database/DeviceManager.java

```diff
@@ -127,8 +144,14 @@ private void updateDeviceCache(boolean force) throws SQLException {
                     devicesByUniqueId.remove(cachedDeviceUniqId);
```

Comment:
Can you please rename `cachedDeviceUniqId` to `cachedDeviceUniqueId`. Not related to this code review, but I just noticed it.